File: examples/python/get_details_from_url.py
from pprint import pprint
from apibara_client import apibara_get

vehicle_url = "https://www.copart.com/lot/51015256/clean-title-2014-cadillac-escalade-esv-platinum-me-windham"
vehicle = apibara_get("/vehicles/urltodetails", {"url": vehicle_url})

pprint(vehicle)

File: examples/node/get-details-from-url.js
import { apibaraGet } from "./apibara-client.js";

const vehicleUrl = process.argv[2] || "https://www.copart.com/lot/51015256/clean-title-2014-cadillac-escalade-esv-platinum-me-windham";
const data = await apibaraGet("/vehicles/urltodetails", { url: vehicleUrl });

console.log(JSON.stringify(data, null, 2));
